stable/prometheus-mysql-exporter/values.yaml

```diff
@@ -70,3 +70,10 @@ cloudsqlproxy:
       "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
       "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/user%40project.iam.gserviceaccount.com"
     }'
+
+# mysql exporter collectors https://github.com/prometheus/mysqld_exporter#collector-flags
+collectors:
```

Comment:
this should be an array to pass to args right ?  then in the deployment could use  `args: {{ args | toyaml }}`

```
args:
  - "--collect.global_status"
  - "--collect.global_variables"
```
or could even do

```
collectors:
  - global_status
  - global_variables
```

and use a helper to add the `--collect.` to the start of each one ?